homeassistant/components/spotify/media_player.py

```diff
@@ -408,45 +423,112 @@ async def async_browse_media(self, media_content_type=None, media_content_id=Non
         return response
 
 
-def build_item_response(spotify, payload):
+def build_item_response(self, payload):
```

Comment:
`self` suggests that it is a class method but `build_item_response` is a normal function. I'd rather use something else like `Spotify` or `api` if you'd want something more generic.